samples/nrf9160/aws_iot/src/main.c

```diff
@@ -216,18 +214,18 @@ static void print_received_data(const char *buf, const char *topic,
 
 	root_obj = cJSON_Parse(buf);
 	if (root_obj == NULL) {
-		printk("cJSON Parse failure");
+		LOG_ERR("cJSON Parse failure");
 		return;
 	}
 
 	str = cJSON_Print(root_obj);
 	if (str == NULL) {
-		printk("Failed to print JSON object");
+		LOG_ERR("Failed to print JSON object");
 		goto clean_exit;
 	}
 
-	printf("Data received from AWS IoT console:\nTopic: %.*s\nMessage: %s\n",
```

Comment:
There were line breaks inside the message which are now gone - is that intentional?